.eslintrc.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable sort-keys */
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```

Comment:
Sort keys wasn't a huge problem in this repo, but it is in the other repos that don't have it. This variation of the rule lets me --fix the sorting stuff, way easier.  

Happy to revert if people are don't on a custom rule though.

Reply:
Specifically interested in @brunobar79's opinion here.